vsintegration/src/FSharp.Editor/Common/CommonHelpers.fs

```diff
@@ -41,6 +41,18 @@ type internal SymbolLookupKind =
     | ByRightColumn
     | ByLongIdent
 
+[<RequireQualifiedAccess>]
+module Option =
+    /// Gets the value associated with the option or the supplied default value.
+    let inline getOrElse v = function
```

Comment:
@KevinRansom It's not your fault. We have to do it because the vsix isn't set up correctly to use the new core functions (see https://github.com/Microsoft/visualfsharp/issues/2063).